Storage/src/StorageObject.php

```diff
@@ -614,12 +626,35 @@ public function downloadToFile($path, array $options = [])
     /**
      * Download an object as a stream.
      *
+     * Please note Google Cloud Storage respects the Range header as specified
+     * by [RFC7233](https://tools.ietf.org/html/rfc7233#section-3.1). See below
+     * for an example of this in action.
+     *
      * Example:
      * ```
      * $stream = $object->downloadAsStream();
      * echo $stream->getContents();
      * ```
      *
+     * ```
+     * // Set the Range header in order to download a subrange of the object. For more examples of
+     * // setting the Range header, please see https://tools.ietf.org/html/rfc7233#section-3.1.
```

Comment:
WDYT of making this a hyperlink? That way it'll be clickable in the docs site.

```suggestion
     * // setting the Range header, please see [RFC7233](https://tools.ietf.org/html/rfc7233#section-3.1).
```